Updated with transcript (#779)
nf-core/nf-co.re: markdown/events/2021/bytesize-14-graphic-design.md

[4:39](https://youtu.be/5jZPucWXnno?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=279) So for nf-core/eager, we decided to make two types of figures. The ones I will go through first serve to explain results, and the second ones are different sorts of pipeline diagrams.

[4:56](https://youtu.be/5jZPucWXnno?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=296) So for the figures in the documentation for `nf-core/eager`, I’m showing you an example of a figure that we have in our manuscript where panel “A” is the actual output from FastQC and panel “B” is the adaptation that we have made. We wanted to make them simple and easy to follow, so we removed a lot of information that was unnecessary for a representative illustration. We also wanted to make them more visually appealing through the use of slightly brighter colours, font types, and soft contours. But we also wanted to make them informative yet accessible, and encourage new users to try using the pipeline on their own. All these figures are available for use under a CC-BY licence, meaning that people can adapt them and use them for their own purposes as well.

> [6:12](https://youtu.be/5jZPucWXnno?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=372) Then the second part is different pipeline diagrams, and what I’ve shown here is a domain overview of the `nf-core/eager` pipeline. We wanted to make it simple and informative, and to be both self-explanatory and complement the documentation. So we have the different inputs marked by these different icons in square boxes to differentiate them from the pipeline elements, and the same with the outputs. We group the different elements of the pipelines in blocks in a way that is easy to follow for individuals running ancient DNA analysis. You can clearly see what happens after you provide your fastq file as an input; the different steps and choices you have at each step etc. The colours here indicate new additions to steps in the pipeline.

Reviewer: ```suggestion
[6:12](https://youtu.be/5jZPucWXnno?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=372) Then the second part is different pipeline diagrams, and what I’ve shown here is the main overview of the `nf-core/eager` pipeline. We wanted to make it simple and informative, and to be both self-explanatory and complement the documentation. So we have the different inputs marked by these different icons in square boxes to differentiate them from the pipeline elements, and the same with the outputs. We group the different elements of the pipelines in blocks in a way that is easy to follow for individuals running ancient DNA analysis. You can clearly see what happens after you provide your fastq file as an input; the different steps and choices you have at each step etc. The colours here indicate new additions to steps in the pipeline.
```

> [7:23](https://youtu.be/5jZPucWXnno?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=443) I’d like to add here that I did not make these from scratch. James mentioned that he liked the pipeline diagram from Sarek, so I took a look and thought he was right. So we picked out elements that we liked, and this is what I suggest that everyone who wants to start making a pipeline diagram does. Look around, remember, you don’t need to reinvent the wheel. You can just pick out things that you like and things you don’t from existing diagrams, and adapt them to make your own.

Reviewer: ```suggestion
[7:23](https://youtu.be/5jZPucWXnno?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=443) I’d like to add here that I did not make these from scratch. James mentioned that he liked the pipeline diagram from Sarek, so I took a look and thought he was right. So we picked out elements that we liked, and this is what I suggest that everyone who wants to start making a pipeline diagram does. Just google, look around, and remember that you don’t need to reinvent the wheel. You can just pick out things that you like and things you don’t from existing diagrams, and adapt them to make your own.
```

> [8:02](https://youtu.be/5jZPucWXnno?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=482) I use Inkscape to make my diagrams, mainly because it is free and because it is intuitive. It doesn’t take long to learn to use and it has a strong online community that makes it easy to troubleshoot or ask for help. There are also online tutorials and DIY blog posts, which are helpful.

Reviewer: ```suggestion
[8:02](https://youtu.be/5jZPucWXnno?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=482) I use Inkscape to make my diagrams, mainly because it is free and because it is intuitive. It doesn’t take long to learn to use and it has a strong online community that makes it easy to troubleshoot or ask for help. You can just google a problem, and there are lots of online tutorials and DIY blog posts, which are helpful.
```

Author: I tried not to refer to google because it isn't accessible everywhere.

Reviewer: Ah fair point. I guess could be replaced with 'search the internet I guess', but not important. Feel free to merge @renbot-bio !

> [8:47](https://youtu.be/5jZPucWXnno?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=527) I also wanted to mention colour schemes. I understand that these are often quite hard to pick. My suggestion is to keep it as simple as possible. I personally think that two-colour schemes are sufficient. For nf-core illustrations, just pick the codes from the logo, you will use the logo in your figure anyway. So if you pick the same colour scheme, you won’t be increasing the number of colours that you use.

Reviewer: ```suggestion
[8:47](https://youtu.be/5jZPucWXnno?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=527) I also wanted to mention colour schemes. I understand that these are often quite hard to pick. My suggestion is to keep it as simple as possible. I personally think that two-colour schemes are sufficient. For nf-core illustrations, just pick the codes from the logo, as you will likely be using the logo in your figure anyway. So if you pick the same colour scheme, you won’t be increasing the number of colours that you use.
```
